apollo-core/src/main/java/com/ctrip/framework/apollo/core/dto/ConfigurationChange.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2022 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.core.dto;
+
+
+/**
+ * Holds the information for a Configuration change.
+ *
+ * @author jason
```

Comment:
We usually omit author information for new files.
```suggestion
 * @since 2.0.0
```